### PR TITLE
Viewer3d order display primitives

### PR DESCRIPTION
**Pull request by kerautret**
Small PR just to change the chronological order to diplay primitives (in the draw function) in order to see see textured image primitives trough the transparency of other 3D primitives.
It fix some a display bug when for instance we display a mesh with transparency in front a slice image.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -125,6 +125,7 @@
   [#1016](https://github.com/DGtal-team/DGtal/pull/1016))
  - Viewer3D: Shift-L / L key binding added to save and restore camera settings. (Bertrand Kerautret,
   [#1024](https://github.com/DGtal-team/DGtal/pull/1024))
+ - Viewer3D:  change the chronological order to diplay primitives (in the draw function) in order to see see textured image primitives trough the transparency of other 3D primitives. (Bertrand Kerautret,  [#1041](https://github.com/DGtal-team/DGtal/pull/1041))
```
Comment (dcoeurjo) [inline ChangeLog.md]: trough -> through
(pelles keep 80col edit)

**Comment by dcoeurjo** [comment]
otherwise this PR is fine. thanks

**Comment by kerautret** [comment]
@dcoeurjo done with aslo other 80 cols corrections.

**Comment by dcoeurjo** [comment]
thanks. merging.

**Pull request merged**